docs/source/changelog.rst

```diff
@@ -11,6 +11,7 @@ Changelog
         * Set index after adding ancestor relationship variables (:pr:`668`)
         * Fix user-supplied variable_types modification in Entity init (:pr:`675`)
         * Don't calculate dependencies of unnecessary features (:pr:`667`)
+        * Normalize entity same index as base entity #681 (:pr:`681`)
```

Comment:
The ":pr:681" syntax will create a link to this pr, you can remove the "#681".  Also can you add a little more to the description so someone unfamiliar could grasp the issue?